chaincode-labs/chaincode-podcast/tracepoints-and-monitoring-the-bitcoin-network.md

```diff
@@ -85,317 +85,317 @@ Hope you enjoy it, too.
-We hope that's a big part, but we can actually see transactions that are missing from the block that we think should be in the block.
+So obviously there are some differences between the my nodes mempool and the mining pools mempool and we can’t exactly time when the mining pool actually created his block template but we can still compare the template, my template and the pools block and figure out, for example, which transactions are shared.
+We hope that’s a big part, but we can actually see transactions that are missing from the block that we think should be in the block.
 We can also see transactions that are extra to the block.
-So one transaction that's always extra is the coinbase.
-We don't have that in our block template.
+So one transaction that’s always extra is the coinbase.
+We don’t have that in our block template.
 Some other times we for example see the payouts and consideration transactions from pools which they sometimes include via a zero fee payment.
-So they don't specify any fees, they are not related on the network and we can detect them being included there.
+So they don’t specify any fees, they are not related on the network and we can detect them being included there.
 And of course transaction accelerators for example ERPTC runs a transaction accelerator, you pay an out-of-band fee and they include your low fee transaction really early on in their block and we can see that for example.
 
-Speaker 1: 00:10:32
+Mark Erhardt: 00:10:32
 
-So you would say that generally you see all the mining pools you're observing as using the same block building as Bitcoin Core?
+So you would say that generally you see all the mining pools you’re observing as using the same block building as Bitcoin Core?
 Yes.
 But they sometimes prioritize transactions because of out of band or their own usage.
 
-Speaker 2: 00:10:45
+0xB10C: 00:10:45
 
 Yeah, I think they use the ERPC prioritized transaction.
 
-Speaker 1: 00:10:49
+Mark Erhardt: 00:10:49
 
-So it's consistent with Bitcoin Core being run by all of them?
+So it’s consistent with Bitcoin Core being run by all of them?
 
-Speaker 2: 00:10:53
+0xB10C: 00:10:53
 
 Yeah, I think so.
-And we don't see too much deviation from the block template actually.
+And we don’t see too much deviation from the block template actually.
 Right.
 
-Speaker 0: 00:10:59
+Adam Jonas: 00:10:59
 
-Well, that's Maybe more fodder for why, Mark, you should be continuing to work on that project.
+Well, that’s Maybe more fodder for why, Mark, you should be continuing to work on that project.
 Yeah, I know.
 
 ## User-space Statically Defined Tracing (USDT)
 
-Speaker 0: 00:11:06
+Adam Jonas: 00:11:06
 
 Cool.
-One of the other projects you've been working on is adding trace points to Bitcoin Core.
+One of the other projects you’ve been working on is adding trace points to Bitcoin Core.
```

Comment:
The correct word "tracepoints" instead of "trace points" is on the title of the transcript, making it easier to catch and fix those.
```suggestion
One of the other projects you’ve been working on is adding tracepoints to Bitcoin Core.
```